core-bundle/tests/Contao/Database/StatementTest.php

```diff
@@ -1,333 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-/*
- * This file is part of Contao.
- *
- * (c) Leo Feyer
- *
- * @license LGPL-3.0-or-later
- */
-
-namespace Contao\CoreBundle\Tests\Contao\Database;
-
-use Contao\CoreBundle\Tests\TestCase;
-use Contao\Database\Statement;
-use Contao\System;
-use Doctrine\DBAL\Connection;
-use Doctrine\DBAL\Driver\Mysqli\Exception\ConnectionError;
-use Doctrine\DBAL\Exception\DriverException;
-use Doctrine\DBAL\Platforms\AbstractPlatform;
-use Doctrine\DBAL\Result;
-use Symfony\Bridge\PhpUnit\ExpectDeprecationTrait;
-use Symfony\Component\DependencyInjection\Container;
-
-class StatementTest extends TestCase
```

Comment:
This should not be removed here IMO, there are tests for the correct behavior in here too (not only tests for deprecations).

Reply:
Which ones? They all contain `$this->expectDeprecation()`.